docs/development/architecture.md

```diff
@@ -1,9 +1,10 @@
 # ServiceX Architecture
 
 ## Frontend
+
  ServiceX presents a RESTful interface for submitting transformation requests and
- requesting status updates. Most users will not wish to interact directly with 
- the REST interface so typically, ServiceX is accessed via clients distributed 
+ requesting status updates. Most users will not wish to interact directly with
```

Comment:
Please be careful about introducing whitespace changes. It makes it hard to see what changed here...

Reply:
my VSCode enforces rules on md files too. I find the changes it makes or proposes usually improve the doc readability. But it gets too pedantic sometimes.